wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_messages.scss

```diff
@@ -44,12 +44,20 @@
     }
 
     .warning {
-        background-color: $color-orange;
+        background-color: $color-button-warning;
+        color: $color-button-warning-text;
 
         &:before {
             font-family: wagtail;
             content: '!';
         }
+
+        .avatar {
+            &:before {
+                color: $color-button-warning-text;
+                border: 1px solid $color-button-warning-text;
+            }
+        }
```

Comment:
On screens where you're looking at a previous revision the avatar and user's avatar name are shown. Currently it's white text on orange background. It's confusing because it's slightly poor name spacing, since the revision screens use the `.error` class throughout to get their style